feat: 마이 페이지 UI 구현(#136)

icw0201 opened the pull request:
## 📌 Issue

<!-- 해결하려는 이슈 번호나 주제를 명확하게 적어주세요. -->

- 관련 이슈: close #136 

---

## 🛠 작업 내용

- 마이페이지에 하드코딩된 더미 데이터를 넣어 UI구성을 마무리했습니다.
<img width="582" height="912" alt="image" src="https://github.com/user-attachments/assets/d39739a0-8dad-447b-b2f9-f7ca166ded84" />
<img width="560" height="664" alt="image" src="https://github.com/user-attachments/assets/8c365891-9f94-4689-ae58-7d7224f1e2ab" />
<img width="480" height="835" alt="image" src="https://github.com/user-attachments/assets/f9a65c3b-ba55-43c5-b98d-3008ca4df423" />

- 남은 할 일
 - 유저 정보 받아와서 페이지에 바르기 (이 부분은 결과 데이터가 정제된 뒤 적용 가능할 것 같아, 구현 순서가 제법 뒤로 밀릴 것으로 예측됩니다.)
 - 헤더 작업자(@Jihye-kr )가 프로필 사진 컴포넌트 구현 완료하면 마이페이지에 적용 
- ***

## 🚀 기타 사항

<!-- 리뷰어가 추가적으로 알아야 할 사항이 있다면 기재해주세요. -->

- 추가적인 내용을 작성해주세요.(참고 자료, 협업 내용)
- @layout-SY 님께 : 주소 드롭다운 컴포넌트의 스타일을 일부 수정했으니 확인 바랍니다.
 - 수정 내역
 - 가로 폭 100%로 설정 (textInput컴포넌트와 같은 양식 사용. 부모 컴포넌트에서 가로 폭을 조절합니다.)
 - 드롭다운 연 뒤 폭 조절(스타일)

- [x] build 체크 했나요? 네.
- [x] dev를 pull 받은 뒤 merge 했나요? 최신 dev에서 분기했기에 지금 merge안해도 됩니다.

icw0201 committed:
refactor: AddressDropDown이 가로 100%영역 차지하도록 정의
- 주로 컴포넌트는 100%차지하고, 부모 컨테이너에서 크기를 조정하는 방식을 차용하고 있었기에
- 해당 컴포넌트에도 적용하여 마이 페이지에서 가로 너비를 예쁘게 맞춥니다.

icw0201 committed:
refactor, style(ui): 주소 드롭다운 여백 줄임
- 실 사용 중 여백이 크게 느껴지기에, 주소 드롭다운 여백 간격을 피그마 수준에 맞춰 줄였습니다.

icw0201 committed:
refactor: ResultAccordion 컴포넌트의 제목 및 숫자 영역 스타일 개선
- title이 길 경우 말줄임 처리

icw0201 committed:
feat: 마이페이지 스타일 업데이트 및 컴포넌트 구조 개선
- 정의된 컴포넌트 활용하여 피그마와 동일하게게 마이페이지 UI 작성
- 하드코딩된 데이터 사용

aiden0413 commented on app/(anon)/mypage/page.styles.ts:
```diff
@@ -1,16 +1,63 @@
+  // 기존 스타일
   mainContainer: "min-h-screen bg-gray-50 py-8",
-  container: "container mx-auto px-4",
+  container: "min-h-screen bg-gray-50 relative",
```
bg-gray-50

icw0201 replied:
수정했습니다

aiden0413 commented on app/(anon)/mypage/page.tsx:
```diff
@@ -1,91 +1,289 @@
-              <p className={styles.tempText}>각 항목별로 확인해야 할 사항들을 정리해드립니다.</p>
-            </div>
-          </ResultAccordion>
+        {/* 가이드 결과 보기 */}
```
가이드 결과 보여주는 부분도 컴포넌트로 분리하고
page에서는 주소선택부분, 등기부등본납세확인서부분, 가이드결과요약(차트부분), 가이드결과보기(아코디언 여러개 있는부분) 이렇게 나누는건 어떨까요

icw0201 replied:
좋은생각이에요
진작 생각하지 못한 제가 원망스럽습니다
훌륭한 피드백 감사합니다
갱신했습니다

icw0201 committed:
fix: 스탭 결과값으로 updated_at이 오지 않던 문제 수정정

icw0201 committed:
feat: 마이페이지 가이드 결과 보기 컴포넌트 추가 및 스타일 개선
- 기존의 하드코딩된 데이터를 mockData로 분리하여 관리
- GuideResultView 컴포넌트를 새로 추가하여 가이드 결과를 표시하도록 구현
- 스타일을 개선하여 UI 일관성 유지

icw0201 committed:
Merge branch 'develop' into feat/#136

Jihye-kr reviewed (approved):
👍 👍 👍 👍 👍 👍
